ambassador/ambassador/envoy/v2/v2listener.py

```diff
@@ -509,6 +510,38 @@ def __init__(self, config: 'V2Config', listener: IRListener) -> None:
                     'format': 'ACCESS [%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\"\n'
                 }
             } ]
+             # Use sane access log spec in JSON
+            if Config.envoy_log_format == "json" :
+                self.access_log = [ {
```

Comment:
Also, instead of setting this twice, I'd put both declaration behind the feature flag; e.g.:
```python
             if Config.envoy_log_format == "json" :
                self.access_log = [ {".... format as defined in this pr ......"}]
             else:
                self.access_log = [ {
                  'name': 'envoy.file_access_log',
                  'config': {
                      'path': '/dev/fd/1',
                      'format': 'ACCESS [%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\"\n'
                  }
                } ]
```

Reply:
done